baremetal.go

```diff
@@ -0,0 +1,114 @@
+// Package vagrantssh provides host connectivity in go for system/integration
+// testing in a multi host environment. It supports two testbed environments viz.
+// baremetal and vagrant
+/*
```

Comment:
yep, I am aware of this documentation. However, the problem is that if I remove the doc string from here I get a golint error. I think that arises from the fact that without this the file starts with copyright notice which golint doesn't like.

```
test -z "$(golint ./... | tee /dev/stderr)"
baremetal.go:1:1: package comment should be of the form "Package vagrantssh ..."
make: *** [golint] Error 1
```

Is there a good way to check what the docks would look like with the current change? may be I can use godoc? let me check and get back on this.